### DISCUSSION TITLE
Run nextflow through TES/Funnel

### DISCUSSION THREAD

**Discussion by javi-gv94** [nextflow-io/nextflow#4280] [Q&A] [answered]
Hi there!

I am trying ro run my nextflow pipeline through TES using Funnel API. However, the only thing I see in the documentation are a few env variables and the usual 'nextflow run...'. Can someone give any clues on how should the API request look? Any example is welcome :)

Thanks a lot

**Comment by bentsherman** [comment, marked as answer]
The current documentation should cover everything you need to run Funnel. You don't need to make any API requests yourself, just provide the URL to your Funnel instance 😄

**Comment by javi-gv94** [comment]
thanks for the information :) I will try that approach.
